### PR TITLE
Receipts: Add `thread_id` to the `/receipt` endpoint

### PR DESCRIPTION
**Pull request by zecakeh**
It seems to have been omitted in #1255.

Per [MSC3771](https://github.com/matrix-org/matrix-spec-proposals/pull/3771).

Preview: https://pr1261--matrix-spec-previews.netlify.app

### CODE REVIEW

File: data/api/client-server/receipts.yaml [hunk 1]
```diff
@@ -88,5 +98,17 @@ paths:
           description: This request was rate-limited.
           schema:
             "$ref": "definitions/errors/rate_limited.yaml"
+        400:
+          description: |-
+            The `thread_id` is not a string, or is empty, or it is provided for
+            a receipt of type `m.fully_read`, or the `event_id` is not related
+            to the `thread_id`.
```
Comment (turt2live) [inline data/api/client-server/receipts.yaml:105]: We can be a bit more relaxed on error code descriptions:
```suggestion
            The `thread_id` is invalid in some way. For example:
            * It is not a string.
            * It is empty.
            * It is provided for an incompatible receipt type.
            * The `event_id` is not related to the `thread_id`.
```

**Review by turt2live — COMMENTED** [review]
oops, thanks for catching! We'll get this included in a patch release.

The changes overall look fairly sane to me. The error code description felt like a run-on sentence, so I've suggested making it a list (while also relaxing the language a bit).

**Review by turt2live — APPROVED** [review]

**Comment by turt2live** [comment]
hmm, this seems to be missing signoff on all the commits - @zecakeh are you able to provide it?

**Comment by zecakeh** [comment]
Signed-off-by: Kévin Commaille <zecakeh@tedomum.fr>

Is this enough?

**Comment by turt2live** [comment]
it is! Thank you.

**Pull request merged**